### ISSUE TITLE
[Feature] Add support for plain URL route matching

### ISSUE THREAD

**Issue by noemi-salaun** [Haehnchen/idea-php-symfony2-plugin#1435]
When using legacy templating solution, you'll find many `href` or form `action` that target plain url. 

This plugin is already capable of finding routes by their Symfony name in twig template but not plain URL in html.

![image](https://user-images.githubusercontent.com/2726492/76614976-5efa2780-6521-11ea-8f1b-b316257df2af.png)

The symfony command `./bin/console router:match /my/route` already provide the right information

![image](https://user-images.githubusercontent.com/2726492/76615130-b39da280-6521-11ea-8db4-a4bcbd1ed4c9.png)

**Comment by Haehnchen** [comment]
yes, its really annoying sometimes, plugin knows must cases, i will try to find if there is any extension point to hide this external warning
